src/main/java/com/netcracker/study/objects/borders/BorderObject.java

```diff
@@ -1,5 +1,5 @@
 package com.netcracker.study.objects.borders;
 
-public interface BorderObject {
-    public void onShoot();
+abstract public class BorderObject implements Shootable, Grenadeable{
```

Comment:
why abstract? it is not used. just implement multiple interface right now

Reply:
I thought of just implementing multiple interfaces by Wall objects, but abstract class is needed to demand every BorderObject (Wall or other future BorderObjects) to implement these interfaces - otherwise you can create a borderobject which does not know what to do when shot upon

Also, it's a way to have a reference type common to all BorderObjects